### DISCUSSION TITLE
Refreshing the digipres.org homepage

### DISCUSSION THREAD

**Discussion by anjackson** [digipres/digipres.github.io#34] [General]
Hi Folks,

The idea behind the https://digipres.org/ homepage still seems like a good one, but I've been wondering if we should change things up a bit to make it more useful and maintainable?

In my opinion, the current site is okay, but has some issues:

- A bit of a clumsy mix of banners, buttons, prose, and lists of links. This makes it a bit difficult to navigate.
- It's hard to keep lists of URLs up to date.
- The Tools section has been superseded by COPTR.
- The Contributions section isn't helpful.
- The Formats section is a bit _too_ hidden away, but also a bit off-topic.

But I'm not sure if/how to change things.

For the IIPC we have an [Awesome List](https://github.com/sindresorhus/awesome/blob/main/awesome.md) instead: https://github.com/iipc/awesome-web-archiving#readme  This works quite well, is easier to modify, and there's tooling to keep things tidy and check for broken links etc.  Much of what we have here could be migrated to Awesome List format, and this page could just present that reformatted, or maybe even just ~~just~~ link to it?

For example, one option would be to keep the banners and buttons at the top,  but move the actual detailed content to a simple Awesome List.  Each of the buttons on the home page would just link to a section of the Awesome List. The formats part could be split out into it's own thing, leaving digipres.org as a very simple gateway page.

Independently from this, I'd also like to pursue the idea of publicising all this more pro-actively. e.g. posting an annual reminder to all the usual mailing lists that this resource exists, and needs updates to stay relevant etc.  This could also mention things like COPTR and https://qanda.digipres.org/ etc.

As this is a community resource, I would like to hear feedback from others in the community.  To start with, I'm going to circulate this to everyone how has contributed to the digipres.org page.

Please let me know what you think.
Andy 

p.s. Reminder circulation: pasig-discuss@lists.stanford.edu digital-preservation@jiscmail.ac.uk dpc-discussion@jiscmail.ac.uk

**Comment by ablwr** [comment]
Agree with your points here, and am in favor of the relative user-friendliness of shifting to an Awesome List format.

**Comment by ross-spencer** [comment]
* +1 awesome list. 
* +1 periodical reminder! (that's a really good idea).

Is it possible to have a github.io page based on an Awesome list and have the banner render still?

**Comment by prwheatley** [comment]
Agree with these suggestions. Awesome looks the way to go

**Comment by anjackson** [comment]
Thanks all.  There was also some helpful feedback on https://digipres.club/web/@anj/109034954378769030 

Overall, the Awesome List seems like a winner, and I think I'll keep the text of the reminder email in the same place.

@ross-spencer I'll look into rendering the Awesome List into the banner page, but I suspect it might make things a bit more fiddly to maintain.

**Comment by anjackson** [comment]
I've attempted to turn this into a list of actions here: https://github.com/digipres/digipres.github.io/issues/37

**Comment by anjackson** [comment]
Okay, we know have a draft of...

- The Awesome List: https://github.com/digipres/awesome-digital-preservation#readme
- The simplified homepage: https://www.digipres.org/
- A reminder email: https://github.com/digipres/awesome-digital-preservation/blob/main/reminder.md#reminder-email

There's more to be done, but I think I'll call this an initial version, and defer the other improvements for later.  The idea would be to send round the reminder soon, and then continue the work on World Digital Preservation Day.

**Comment by anjackson** [comment]
Well, I've been patching up the broken links and I've sent out the reminder. 

Could still use feedback on https://github.com/digipres/awesome-digital-preservation/issues/2 _Review the Awesome List structure, scope and content_

**Comment by anjackson** [comment]
This refresh was done so I'm closing this thread.
